### ISSUE TITLE
No of Epochs in cls modelnet40

### ISSUE THREAD

**Issue by merazlab** [AnTao97/dgcnn.pytorch#32]
Thanks,
I train the same network but not gating 93.3% accuracy. Tell me How many epochs needed for that. Also, what is the use of T Networks?

**Comment by antao97** [comment]
Just use the provided code and use the default settings. You can try to re-run the exp several times to get a better result.

**Comment by merazlab** [comment]
Thank you. For Quick response...
